src/linqmacro.jl

```diff
@@ -62,6 +62,9 @@ julia> @linq df |>
 ```
 """
 macro linq(arg)
+
+    @warn "@linq is deprecated. Use @chain instead. See ? @chain for details"
```

Comment:
`depwarn` doesn't work in macros?
Also:
```suggestion
    @warn "@linq is deprecated. Use @chain instead. See ?@chain for details"
```

Reply:
Do you confirm that `depwarn` cannot be used here?

Reply:
It can. But it says that the warning stems from `eval` at `boot.jl`, which I guess is fine.